### ISSUE TITLE
comment-format complains on second line of comment

### ISSUE THREAD

**Issue by andy-hanson** [palantir/tslint#2840]
### Bug Report

- __TSLint version__: master
- __TypeScript version__: 2.4.0-dev.20170528
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
// This comment
// is long
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "comment-format": [true, "check-uppercase"]
  }
}
```

#### Actual behavior

`ERROR: 2:3  comment-format  comment must start with uppercase letter`

#### Expected behavior

No error.

**Comment by ajafff** [comment]
duplicate of #649
